src/client/app/App.react.js

```diff
@@ -21,6 +22,10 @@ export default class App extends Component {
     users: PropTypes.object.isRequired
   }
 
+  state = {
```

Comment:
We cannot, because then you render the DevTools on client in the first render run and it will discard what was rendered on server.